doc/source/release/1.26.0-notes.rst

```diff
@@ -4,9 +4,64 @@
 NumPy 1.26.0 Release Notes
 ==========================
 
-The NumPy 1.26.0 release is a continuation of the 1.25.x release cycle, but
-with the distutils based build replaced by meson in order to work with Python
-3.12.
+The NumPy 1.26.0 release is a continuation of the 1.25.x release cycle with the
+addition of Python 3.12.0 support. Python 3.12 dropped distutils, consequently
+supporting it required finding a replacement for the setup/distutils based
+build system NumPy was using. We have chosen to use the Meson build system
+instead, and this is the first NumPy release supporting it. We have also moved
+to using the newly released Cython 3.0+. Supporting those two upgrades was a
+large project, over 100 files have been touched in this release. The changelog
+doesn't capture the full extent of the work, special thanks to Ralf Gommers,
+Sayed Adel, Stéfan van der Walt, and Matti Picus who did much of the work in
+the main development branch.
+
+The highlights of this release are:
+
+- Python 3.12.0 support.
+- Cython 3.0.0 compatibility.
+- Use of the Meson build system
+- Updated SIMD support
+- Versioneer replaced
```

Comment:
I'd remove this line, that's something no user or packager should care about either way.